providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/pod_generator.py

```diff
@@ -355,38 +355,12 @@ def construct_pod(
             containers=[main_container],
         )
 
-        if content_json_for_volume:
+        if content_json:
             import shlex
 
-            input_file_path = "/tmp/execute/input.json"
-            execute_volume = V1Volume(
-                name="execute-volume",
-                empty_dir=V1EmptyDirVolumeSource(),
-            )
-
-            execute_volume_mount = V1VolumeMount(
-                name="execute-volume",
-                mount_path="/tmp/execute",
-                read_only=False,
-            )
-
-            escaped_json = shlex.quote(content_json_for_volume)
-            init_container = k8s.V1Container(
-                name="init-container",
-                image="busybox",
-                command=["/bin/sh", "-c", f"echo {escaped_json} > {input_file_path}"],
-                volume_mounts=[execute_volume_mount],
-            )
-
-            main_container.volume_mounts = [execute_volume_mount]
+            escaped_json = shlex.quote(content_json)
             main_container.command = args[:-1]
-            main_container.args = args[-1:]
-
-            podspec = k8s.V1PodSpec(
-                containers=[main_container],
-                volumes=[execute_volume],
-                init_containers=[init_container],
-            )
+            main_container.args = escaped_json
```

Comment:
```suggestion
            main_container.args = [escaped_json]
```
should be a list if i'm not mistaken
https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1Container.md

Reply:
You are indeed correct.

Reply:
Yes correct